File: MuseumAPP/MuseumApp.Core/Entities/Article.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MuseumApp.Core.Entities
{
    public class Article : BaseEntity
    {
        [Required]
        [StringLength(500)]
        public string? Name{get;set;}

        [Required]
        [StringLength(1000)]
        public string? Description{get;set;}

        [Required]
        public bool? Damaged{get;set;}

        [Required]
        public int MuseumId{get;set;}

        [Required]
        public Museum? Museum{get;set;}

    }
}

File: MuseumApp-UI/src/app/models/Article.ts
import { Museum } from "./Museum";

export class Article{
  id?: number;
  name:string;
  description: string;
  damaged: boolean;
  museumId:number;

  constructor(){
    this.name = '';
    this.description = '';
    this.damaged = false;
    this.museumId = 0;
  }
}
